src/platform/wearables/wearable.ts

```diff
@@ -18,19 +18,22 @@ import { WithRequired } from '../../misc'
 /** @alpha */
 export type Wearable = DisplayableDeployment & {
   id: string
-  descriptions: I18N[]
-  collectionAddress?: string
-  rarity?: Rarity
-  names: I18N[]
+  name: string
+  description: string
   data: {
     replaces: WearableCategory[]
     hides: WearableCategory[]
     tags: string[]
     representations: WearableRepresentation[]
     category: WearableCategory
   }
+  i18n: I18N[]
   thumbnail: string
   image: string
+  createdAt: number
+  updatedAt: number
```

Comment:
Would you mind removing `createdAt` and `updatedAt`? We're not using those properties.